integration-tests/main/src/main/resources/META-INF/persistence.xml

```diff
@@ -19,8 +19,10 @@
 
             <!-- Tuning and debugging -->
             <property name="hibernate.connection.pool_size" value="2"/>
-            <property name="hibernate.show_sql" value="false"/>
+            <property name="hibernate.show_sql" value="true"/>
```

Comment:
Would this slow down the CI jobs by producing more output?

Reply:
@kenfinnigan good catch. This should not be changed.